common/lib/container-definitions/package.json

```diff
@@ -42,7 +42,7 @@
   "dependencies": {
     "@fluidframework/common-definitions": "^0.20.1",
     "@fluidframework/core-interfaces": "^0.43.1000",
-    "@fluidframework/driver-definitions": "^0.46.1000",
+    "@fluidframework/driver-definitions": "^0.47.1000-0",
```

Comment:
don't we also need to bump the major version in this package

Reply:
I was not aware of this if so. If increasing the major version is required for changes like this, I believe I'll also need to increase the version for `driver-definitions`.
My understanding was that I would push a change in this package as a pre-release so any future builds won't break based on my changes in the `driver-definitions` package. Then, later on, someone (based on the plan of updating the package versions) would go in and bump the version everywhere.